src/common/utils/validators.py

```diff
@@ -143,6 +143,8 @@ def _validate_entity(
             _validate_primitive_attribute(
                 attributeDefinition, entity[attributeDefinition.name], f"{key}.{attributeDefinition.name}"
             )
+        elif attributeDefinition.attribute_type == "any" and attributeDefinition.name == "default":
```

Comment:
```suggestion
        elif attributeDefinition.attribute_type == "any"
```
Do we only allow for "any" if the attribute is named "default"?

Reply:
My thinking here was that we only want to allow "any" to a type for the "default" attribute so that users cannot use the "any" type when creating models.

Without going too much into the code, I would assume the user can manage skip the validation if they define blueprints with attributeType any. For example, I don't think the aValue attribute in the following blueprint would be validated if attributeType is set to "any"

```json
{
  "type": "dmss://system/SIMOS/Blueprint",
  "extends": ["dmss://system/SIMOS/NamedEntity"],
  "name": "ExampleUserBlueprint",
  "attributes": [
    {
      "name": "aValue",
      "attributeType": "any",
      "type": "dmss://system/SIMOS/BlueprintAttribute",
      "default": [1,2,3]
    }
  ]
}

```